apkid/rules/dex/packers.yara

```diff
@@ -444,3 +444,47 @@ rule jsonpacker : packer
      and ($algo or $algo2 or $algo3)
      and $dexclass
 }
+
+rule multidexpacker : packer
+{
+  meta:
```

Comment:
Would you add the blog post in the `url` metadata field?  https://cryptax.medium.com/multidex-trick-to-unpack-android-bianlian-ed52eb791e56